src/resources/postcss/components/skeleton/buttons/_icon.pcss

```diff
@@ -30,60 +30,36 @@
 	 * ----------------------------------------------------------------------------- */
 
 	/* -----------------------------------------------------------------------------
-	 * Button: Icon Caret Left
+	 * Button: Icon Caret Left/Right
 	 * ----------------------------------------------------------------------------- */
 
-	.tribe-common-c-btn-icon--caret-left {
-
-		&:before {
-			background-image: svg-inline(caret-left-secondary);
-			height: 20px;
-			width: 12px;
-		}
-
+	.tribe-common-c-btn-icon--caret-left,
+	.tribe-common-c-btn-icon--caret-right {
+		
 		&:active,
 		&:hover,
 		&:focus {
 
-			&:before {
-				background-image: svg-inline(caret-left);
+			.tribe-common-c-btn-icon-svg path {
+				fill: var(--color-icon-primary);
 			}
 		}
 
 		&:disabled {
 
-			&:before {
-				background-image: svg-inline(caret-left-disabled);
+			.tribe-common-c-btn-icon-svg path {
+				fill: var(--color-icon-disabled);
 			}
 		}
 	}
-
-	/* -----------------------------------------------------------------------------
-	 * Button: Icon Caret Right
-	 * ----------------------------------------------------------------------------- */
-
-	.tribe-common-c-btn-icon--caret-right {
-
-		&:before {
-			background-image: svg-inline(caret-right-secondary);
-			height: 20px;
-			width: 12px;
-		}
-
-		&:active,
-		&:hover,
-		&:focus {
-
-			&:before {
-				background-image: svg-inline(caret-right);
-			}
-		}
-
-		&:disabled {
-
-			&:before {
-				background-image: svg-inline(caret-right-disabled);
-			}
+	
+	.tribe-common-c-btn-icon-svg--caret-left,
+	.tribe-common-c-btn-icon-svg--caret-right {
+		height: 19px;
+		width: 11px;
+		
+		path {
+			fill: var(--color-icon-secondary);
 		}
 	}
```

Comment:
Since we already have the modifier on the `.tribe-common-c-btn-icon` class, let's move this under `.tribe-common-c-btn-icon--caret-left, .tribe-common-c-btn-icon--caret-right` above, change the class to `.tribe-common-c-btn-icon__svg`, and remove the height.